### PR TITLE
Pricing per user

### PR DESCRIPTION
**Pull request by Skyriixxx**

### CODE REVIEW

File: src/authorization/AuthorizationsDefinition.ts [hunk 1]
```diff
@@ -1471,7 +1471,7 @@ export const AUTHORIZATION_DEFINITION: AuthorizationDefinition = {
         ],
       },
       {
-        resource: Entity.USER, action: Action.READ,
+        resource: Entity.USER, action: [Action.READ, Action.MAINTAIN_PRICING_DEFINITIONS],
```
Comment (LucasBrazi06) [inline src/authorization/AuthorizationsDefinition.ts:1474]: MAINTAIN_PRICING_DEFINITIONS should be with UPDATE and not READ.

**Review by LucasBrazi06 — CHANGES_REQUESTED** [review]
Check my comments

File: src/authorization/AuthorizationsDefinition.ts (hunk 1 above)
Comment (Skyriixxx) [reply src/authorization/AuthorizationsDefinition.ts]: Done

**Review by LucasBrazi06 — APPROVED** [review]
Looks good!

**Pull request merged**